### PR TITLE
Fix grids and typographys

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/base/_unclassed.scss [hunk 1]
```diff
@@ -20,4 +20,15 @@ a {
...
-p { color: $light-purple; }
\ No newline at end of file
+p {
+  color: $light-purple;
+
+  > a {
+    border-bottom: 2px solid $selected;
+    padding-bottom: 2px;
+    &:hover { color: #fff; }
+
```
Comment (ghost) [inline src/base/_unclassed.scss]: Why the extra space?

File: src/components/GraphPage/index.js [hunk 2]
```diff
@@ -27,21 +27,23 @@ class GraphPage extends Component {
       <section id="graph-page">
         <h3>wonderful women</h3>
         <div className="content">
-          <p>{ `There's a woman behind every single change in our history — whether you know her
-          or not. We have picked a few (for now) to try and show it, by sorting them into their
-          influence areas and through time.` }</p>
+          <p>{ `There's a woman behind every single change in our history — whether we know her or not. We have picked a few to try and show it, by sorting them by their influence fields and through time. Take a look and get surprised by how much girl power history can reveal!` }</p>
```
Comment (ghost) [inline src/components/GraphPage/index.js]: Please, break this line

File: src/components/GraphPage/index.js [hunk 3]
```diff
@@ -27,21 +27,23 @@ class GraphPage extends Component {
...
+              <h3 className={ this.styleTitle(this.props.titlePosition) }>
+                Who are the wonderful women who helped shape the world?
+              </h3>
+              <div className="legend">
+                <span>Women</span>
+              </div>
+            </div>
+            <DataLoader fetchData={this.fetchData} />
+            <FilterableScatterPlot dataList={this.state.dataList} options={this.state.options} />
+            <p className="information">Information: We collected the information presented from public sources like <a href="https://www.wikipedia.org/" target="_blank">Wikipedia</a> and do not take responsability for the content in the external links.</p>
```
Comment (ghost) [inline src/components/GraphPage/index.js]: Please, break this line too.

Comment (ghost) [reply src/components/GraphPage/index.js]: Also, use the template standard we've been using for long lines of text. (Look at line 30 of `src/components/GraphPage/index.js`)

File: src/components/Share/style.scss [hunk 4]
```diff
@@ -6,28 +6,56 @@
...
-    @include grid(8);
-    @include offsetLeft(2);
-    @include offsetRight(2);
+    @include wrap-mobile;
+    @include clearfix;
+
+    @include large-up {
+      @include grid(8);
+      @include offsetLeft(2);
+      @include offsetRight(2);
```
Comment (ghost) [inline src/components/Share/style.scss]: If you're using `grid(8)` and `offsetLeft(2)`, is there a need for `offsetRight`?

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by Yurickh — APPROVED** [review]
Very good sir

**Pull request merged**